### PR TITLE
Add Audacity Alpha version 3.0.3-RC2

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/20353)

### CODE REVIEW

File: manifests/a/Audacity/Audacity/Alpha/3.0.3-RC2/Audacity.Audacity.Alpha.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
+PackageIdentifier: Audacity.Audacity.Alpha
+PackageVersion: 3.0.3-RC2
```
Comment (ItzLevvie) [inline manifests/a/Audacity/Audacity/Alpha/3.0.3-RC2/Audacity.Audacity.Alpha.locale.en-US.yaml:3]:
```suggestion
PackageVersion: 3.0.3
```

File: manifests/a/Audacity/Audacity/Alpha/3.0.3-RC2/Audacity.Audacity.Alpha.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+PackageIdentifier: Audacity.Audacity.Alpha
+PackageVersion: 3.0.3-RC2
```
Comment (ItzLevvie) [inline manifests/a/Audacity/Audacity/Alpha/3.0.3-RC2/Audacity.Audacity.Alpha.installer.yaml:3]:
```suggestion
PackageVersion: 3.0.3
```

File: manifests/a/Audacity/Audacity/Alpha/3.0.3-RC2/Audacity.Audacity.Alpha.yaml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+PackageIdentifier: Audacity.Audacity.Alpha
+PackageVersion: 3.0.3-RC2
```
Comment (ItzLevvie) [inline manifests/a/Audacity/Audacity/Alpha/3.0.3-RC2/Audacity.Audacity.Alpha.yaml:3]:
```suggestion
PackageVersion: 3.0.3
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/a/Audacity/Audacity/Alpha/3.0.3-RC2/Audacity.Audacity.Alpha.locale.en-US.yaml (hunk 1 above)
Comment (ItzLevvie) [reply manifests/a/Audacity/Audacity/Alpha/3.0.3-RC2/Audacity.Audacity.Alpha.locale.en-US.yaml:3]: Please update the version folder as well using GitHub Desktop.

**Comment by vedantmgoyal2009** [comment]
There are two versions 3.0.3-RC1 and 3.0.3-RC2. That's why I had made it 3.0.3-RC2.

**Comment by ItzLevvie** [comment]
**You'll have to update the version folder as well.**

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**